src/main/java/com/fasterxml/jackson/core/json/async/NonBlockingJsonParserBase.java

```diff
@@ -99,6 +99,8 @@
 
     protected final static int MINOR_VALUE_TOKEN_NON_STD = 19;
 
+    protected final static int MINOR_NUMBER_PLUS = 230;
```

Comment:
How about use `22` instead since that's available?

Reply:
changed